CMakeLists.txt

```diff
@@ -22,7 +22,7 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
 
 option(BUILD_GSTREAMER_PLUGIN "enable gstreamer plugin" "OFF")
 
-option(BUILD_ROS_INTERFACE "Enable building ROS dependent plugins" "OFF")
+option(BUILD_ROS_INTERFACE "Enable building ROS dependent plugins" "ON")
```

Comment:
This should be defaulted to OFF

Reply:
> This should be defaulted to OFF

Then how can i force CMake to build ROS plugin? It there any option to enable it on higher CMake level?

Reply:
This is a Cmake option. So you would build like `catkin build -DBUILD_ROS_INTERFACE=ON`.